Makefile

```diff
@@ -156,7 +156,7 @@ test-e2e: $(GINKGO) $(KIND) ## Run e2e tests
 
 .PHONY: test-e2e-new ## Run new e2e tests using clusterctl
 test-e2e-new: $(GINKGO) $(CONFORMANCE_CI_TEMPLATE) $(KIND) $(SSM_PLUGIN) e2e-image ## Run e2e tests
-	time $(GINKGO) -trace -progress -nodes=$(GINKGO_NODES) -v -tags=e2e -focus=$(E2E_FOCUS) $(GINKGO_ARGS) ./test/e2e_new/... -- -config-path="$(E2E_CONF_PATH)" -artifacts-folder="$(ARTIFACTS)" $(E2E_ARGS)
+	time $(GINKGO) -trace -progress -v -tags=e2e -focus=$(E2E_FOCUS) $(GINKGO_ARGS) ./test/e2e_new/... -- -config-path="$(E2E_CONF_PATH)" -artifacts-folder="$(ARTIFACTS)" $(E2E_ARGS)
```

Comment:
One potential option that we may have rather than trying to force serialization for everything, we could follow the upstream convention of marking specs that need to be serialized with [Serial], then we could have two separate make targets, which set the focus appropriately to either target [Serial] specs or exclude [Serial] specs.

Reply:
I originally went that route as well but then realized that none of the tests can be run in parallel since they all create at least 1 cluster and creating 2 clusters would surpass the resource limit.

Reply:
fwiw, ginkgo recommend separate packages for serial vs. parallel. Either that or an external lock.